doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -351,16 +379,33 @@ def _collect_outliers(self):
                 continue
 
             dgk = image_meta.distgit_key
-            package_name = image_meta.get_component_name()
-            basis_event_dict = image_meta.get_latest_brew_build(default=None, complete_before_event=self.basis_event)
-            if not basis_event_dict:
-                self._exit_with_error(f'No image was found for assembly {self.runtime.assembly} for component {dgk} '
-                                      f'at estimated brew event {self.basis_event}. No normal reason for this to '
-                                      f'happen so exiting out of caution.')
 
-            basis_event_build_dict: BrewBuildRecordInspector = BrewBuildRecordInspector(
-                self.runtime, basis_event_dict['id'])
-            basis_event_build_nvr = basis_event_build_dict.get_nvr()
+            if self.runtime.build_system == 'brew':
+                package_name = image_meta.get_component_name()
+                basis_event_dict = image_meta.get_latest_brew_build(default=None, complete_before_event=self.basis_event)
+                if not basis_event_dict:
+                    self._exit_with_error(f'No image was found for assembly {self.runtime.assembly} for component {dgk} '
+                                          f'at estimated brew event {self.basis_event}. No normal reason for this to '
+                                          f'happen so exiting out of caution.')
+
+                basis_event_build_dict: BrewBuildRecordInspector = BrewBuildRecordInspector(
+                    self.runtime, basis_event_dict['id'])
+                basis_event_build_nvr = basis_event_build_dict.get_nvr()
+
+            else:
+                package_name = dgk
+                build_record = await anext(self.runtime.konflux_db.search_builds_by_fields(
```

Comment:
Can we use `meta.get_latest_konflux_build` here? is it `konflux_event` aware?

Reply:
Yes! Done it with 8f5e4b8f76fbbb13dd4f8b9473023c31f83cc1ac :+1: 

Note that konflux event is not a BigQuery concept. We store timestamps, but in order for us to reuse most of gen-assembly existing logic I had to introduce this timestamp-to-integer conversion.